AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -7776,7 +7776,7 @@ insideevs.com,motor1.com##.m1-footer-messages
 polityka.pl##.md-cookie-warning
 mini.hu,mini.com.tr,mini.de##.md-cookiesoptinout
 aws.training,aws.amazon.com##.mo-sitePriv
-myanimelist.net##.modal-content
+inps.it,myanimelist.net##.modal-content
```

Comment:
Unfortunately, it's not enough. Page scrolling is blocked, so I added a fix directly.